wazimap_ng/profile/serializers/metrics_serializer.py

```diff
@@ -60,7 +61,12 @@ def MetricsSerializer(profile, geography, version):
                             "key_metrics": [{
                                 "label": profile_key_metric.label,
                                 "value": val,
-                                "method": denominator
+                                "method": denominator,
+                                "metadata": {
```

Comment:
@KhadijaMahanga It will b good if we can use metadata serializer here: https://github.com/OpenUpSA/wazimap-ng/blob/staging/wazimap_ng/datasets/serializers.py#L77
Thanks :)